conferences/abstracts/OSF19_Demo_submission.md

```diff
@@ -0,0 +1,91 @@
+# OSFair2019 Demo Submission 
+Open Science Fair 2019, September 16-18, Porto, Portugal
+
+*For Authors: remove text in italics and replace with your own content.*
+
+## The Turing Way: A handbook for reproducible data science 
+
+*Author 1 name, affiliation, email address; Author 2 name, affiliation, email address*
+
+The Turing Way, Alan Turing Institute; Rachael Ainsworth, University of Manchester, rachael.ainsworth@manchester.ac.uk
```

Comment:
I don't mind you being the author for a demo presentation @rainsworth. Up to you if you want to leave the Turing Way in, but - for example - when I give talks I just have myself as the "author".

See, for example: https://github.com/alan-turing-institute/the-turing-way/blob/master/conferences/abstracts/ukoswg_submission.md#authors